locale/en/manager.po

```diff
@@ -1965,6 +1965,17 @@ msgstr "Assign DOIs to all published journal objects that have not been assigned
 msgid "doi.manager.settings.doiAssignJournalWide.confirm"
 msgstr "Are you sure you wish to assign DOIs to all published objects that have not been assigned DOIs?"
 
+msgid "doi.manager.settings.doiVersioning.description"
+msgstr ""
+"Assign a new DOI to each publication version? Most journals will "
```

Comment:
I wonder if this could be something like `Most users will` rather than `Most journals/presses/etc` so it can be in pkp-lib instead. What do you think?

Reply:
Yes, that make sense for OJS and OMP. OPS has a different text and 'yes' as default option. I think if there is a OPS specific translation, it will be taken, so yes, I will move this to pkp-lib.